src/components/Wizard/CreateVmWizard/tests/__snapshots__/BasicSettingsTab.test.js.snap

```diff
@@ -96,19 +96,58 @@ exports[`<BasicSettingsTab /> for Create VM Template renders correctly 1`] = `
       },
       "operatingSystem": Object {
         "choices": Array [
-          "fedora29",
-          "fedora28",
-          "fedora27",
-          "fedora26",
-          "fedora25",
-          "fedora24",
-          "fedora23",
-          "rhel7.0",
-          "ubuntu18.04",
-          "win2k12r2",
-          "win2k8r2",
-          "win2k8",
-          "win10",
+          Object {
```

Comment:
NACK from me.

This needs to be parsed from some libosinfo source.

/cc @MarSik

Reply:
Yes, the templates get the OS names from libosinfo. See here: https://github.com/kubevirt/common-templates/blob/master/templates/_linux.yaml#L17

Reply:
My point is: The array you create in the code, this should not happen, instead you should bundle the libosinfo db and parse teh contents from there. IMO

Reply:
Fabian, those are **test data**. You do not want to depend on 3rd party libraries there, the test inputs and outputs must be 100% predictable.